Source/Additional Samples/Clima_SQLite_Demo/Clima_SQLite_Demo.csproj

```diff
@@ -10,7 +10,7 @@
     <LangVersion>9.0</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Meadow.Foundation.Web.Maple.Server" Version="1.6.0" />
+    <PackageReference Include="Meadow.Foundation.Web.Maple.Server" Version="1.7.0" />
     <ProjectReference Include="..\..\Meadow.Clima\Meadow.Clima.csproj" />
     <PackageReference Include="System.Text.Json" Version="*" />
```

Comment:
This is a wildcard version we overlook, System.Text.Json 8.0 wont work with Meadow, we need to hardcode it to 7.0.4